### PR TITLE
We need to be able to configure extra grafana Prometheus datasources

### PR DESCRIPTION
**Pull request by MattiasGees**
This adds the possibility to configure extra Prometheus datasources besides the default one. This is handy if we have multiple proemtheus instances running and we want to only use one grafana installation.

### CODE REVIEW

File: templates/helm-values-grafana.tpl.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dataSource:  
+  prometheus-datasource.json: |+
```
Comment (SamClinckspoor) [inline templates/helm-values-grafana.tpl.yaml]: won't this create a duplicate source?

Comment (MattiasGees) [reply templates/helm-values-grafana.tpl.yaml]: Can you elaborate what you mean?

Comment (SamClinckspoor) [reply templates/helm-values-grafana.tpl.yaml]: currently the infrastructure prometheus is already a datasource. won't setting this create 2 datasources of the same endpoint?

Comment (MattiasGees) [reply templates/helm-values-grafana.tpl.yaml]: No from the moment you create one datasource for helm grafana, you have to declare them all.
See [this](https://github.com/coreos/prometheus-operator/blob/master/helm/grafana/templates/dashboards-configmap.yaml#L16-L27) if..else structure

**Pull request merged**